srcflib/email/__init__.py

```diff
@@ -87,7 +87,7 @@ def __exit__(self, exception_type, exception_value, traceback):
 DEFAULT_WRAPPER = EmailWrapper(subject="[SRCF] {}")
 
 
-def send(target: Owner, template: str, context: dict=None, session: SQLA_SESSION=None):
+def send(target: Owner, template: str, context: dict = None, session: SQLA_SESSION = None):
```

Comment:
Should `SQLA_SESSION` be capitalised this way if it's just a class?  I would prefer `SqlaSession` or `SQLASession` instead.

Reply:
Alternatively, `sqla_Session` or `sqlalchemy_Session` to match how we import `srcf.database.schema.Job` as `db_Job` in srcf/controllib/jobs.py

Reply:
Let's go with `SQLASession` and `RequestsSession`.